playbooks/roles/nginx/templates/assets.j2

```diff
@@ -26,6 +26,14 @@ server {
         proxy_pass $agreements_s3_url;
     }
 
+    # Hack to force g7 communications to the old bucket to preserve upload times
+    location ^~ /g-cloud-7-updates/communications {
```

Comment:
Other rules in this file have a space between the `^` and `~` - I don't know enough about nginx config to say if that is significant at all.